modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -1,10 +1,10 @@
 {
-  "addColumnBefore": "Add Column Before",
   "addColumnAfter": "Add Column After",
+  "addColumnBefore": "Add Column Before",
```

Comment:
I've sorted the entries A-Z.
There are 2 additions on all languages
* `"relationshipSuggestionHelp": "Type to search all {{ type }}...",`
* `"relationshipSuggestionLabel": "Suggestions",`